addons/point_of_sale/tools/posbox/configuration/upgrade.sh

```diff
@@ -79,13 +93,26 @@ download_raspios () {
             echo "Error_Raspios_Download"
             exit 0
         fi
-        unzip "${RASPIOS}"
+        pkill chromium # kill chromium to free space
+        rm -rf /tmp/0  # clean chrome workspace to free space
+        printf "Uncompressing %s..." "${RASPIOS}"
+        touch .uncompressing
+        xz -d -T4 "${RASPIOS}"  # uncompressing raspios.xz w/ 2 threads
```

Comment:
`xz` sometimes slowing, even blocking (even with the `-T4` giving it 4 threads)...